### PR TITLE
PR Comments, CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by DerekRoberts**
Provide thanks and directions on new PRs. Follow-up merges with another message. Added a really basic CONTRIBUTING.md, which will link properly after merge to master.

Part of GitHub Actions pipeline rollout 2 of 2.
bcgov/wps#235

### CODE REVIEW

**Comment by bcgov-wps** [comment]
Thanks for the pull request! <br> Please see [CONTRIBUTING.md](https://github.com/bcgov/wps-web/blob/master/CONTRIBUTING.yaml) \ for details.

**Comment by bcgov-wps** [comment]
Thanks for the pull request! <br> Please see [CONTRIBUTING.md](https://github.com/bcgov/wps-web/blob/master/CONTRIBUTING.yaml)  for details.

**Comment by bcgov-wps** [comment]
Thanks for the pull request! <br> Please see [CONTRIBUTING.md](https://github.com/bcgov/wps-web/blob/master/CONTRIBUTING.yaml)  for details.

**Comment by bcgov-wps** [comment]
Thanks for the pull request! <br> Please see [CONTRIBUTING.md](https://github.com/bcgov/wps-web/blob/master/CONTRIBUTING.yaml)  for details.

**Comment by bcgov-wps** [comment]
This pull request has been merged! <br> ACTION: Please ensure any post-merge steps have been taken.

File: .github/workflows/comments.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        with:
+          message: >-
+            Thanks for the pull request! <br>
+            Please see [CONTRIBUTING.md](https://github.com/bcgov/wps-web/blob/master/CONTRIBUTING.yaml) 
+            for details.
+          GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+
+      - name: Comment on PR
+        if: github.event.pull_request.merged == true
+        uses: thollander/actions-comment-pull-request@master
```
Comment (Sybrand) [inline .github/workflows/comments.yml:29]: @Kyubinhan @DerekRoberts @andrea-williams something to ruminate on: using github actions from various repos - it implies a level of trust. I think we need to educate ourselves on the implications - a quick google will take you down a rabbit hole: https://julienrenaux.fr/2019/12/20/github-actions-security-risk/

**Pull request merged**